.appveyor.yml

```diff
@@ -86,34 +93,37 @@ environment:
       B2_ADDRESS_MODEL: 32 # No 64bit support
 
     - FLAVOR: cygwin (32-bit)
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
       ADDPATH: C:\cygwin\bin;
+      ADDCOMMANDS: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars32.bat"'
```

Comment:
In the case of clang-cl, the ADDCOMMANDS was precisely the fix.  The build was broken, couldn't find mt.exe, and after that change it succeeded.
The question is about cygwin though.   
Testing a few repos now.

Reply:
Yes, but you added it to cygwin, not clang-cl which IMO is definitely wrong. Why would you want to setup MSVC dev tools for using cygwin? And previously it didn't seem to be required. Looks like a bug somewhere else too me, as e.g. b2 does this already. We shouldn't need to do this here. Maybe remove all of those to let it fail on CI then investigate further?

Cygwin doesn't seem to search to files folder (anymore?), see Slack. Although I'm surprised this has worked before.
I'd say we need some more tests... Could you extract the test case (the include thing) into a new PR? So we can check/verify it works with current appveyor configs and if not fix that first. Then update. Otherwise there are to many things changing at once.

Reply:
> extract the test case 

done

> remove all of those to let it fail

Alright, pushed the change to let jobs fail. Let's see.